src/ONYXKEYS.js

```diff
@@ -110,6 +110,7 @@ export default {
         POLICY: 'policy_',
         REPORT_IS_COMPOSER_FULL_SIZE: 'reportIsComposerFullSize_',
         POLICY_MEMBER_LIST: 'policyMemberList_',
+        WORKSPACE_INVITE_MEMBERS_DRAFT: 'workspaceInviteMembersDraft_',
```

Comment:
Why is this key a collection?

Reply:
This can have values for multiple workspaces if the user ended the invite process in the invite message screen. Hence a collection would be best in this case similar to REPORT_DRAFT_COMMENT - https://github.com/Expensify/App/blob/c6926f1d0e07248f89d362f829f4a9dc6a302e5f/src/ONYXKEYS.js#L107

Reply:
makes sense. thanks for clarifying!